pkg/symbolizer/addr2liner.go

```diff
@@ -0,0 +1,184 @@
+// Copyright 2016 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+// TODO: strip " (discriminator N)", "constprop", "isra" from function names.
+
+package symbolizer
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"os/exec"
+	"strconv"
+	"strings"
+
+	"github.com/google/syzkaller/pkg/osutil"
+	"github.com/google/syzkaller/sys/targets"
+)
+
+type addr2Liner struct {
```

Comment:
Nit: AFAIK the `-er` suffix is supposed to be used for interface names in Go. Maybe just `addr2Line`?

Reply:
Symbolizer, compiler, merger, fuzzer, manager, serializer...
It resonates with "symbolizer"... This is the main reason for this name. The plan is to add also "dwarfer.go"...
Ok to rename if it sounds weird to you.

Reply:
`addr2Liner` and `dwarfer` definitely sound weird to me.

Reply:
ptal